app/dashboard/helpers.py

```diff
@@ -504,6 +504,41 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
                 except Exception as e:
                     logger.error(e)
 
+            bounty_invitees = metadata.get('invite', '')
```

Comment:
I think we need some kind of check here to make sure its actually a new bounty, or that the invitee hasn't already been emailed about this bounty, since this function will get called each time the bounty is updated.

Reply:
True. Will add that check.

Reply:
Done